ui/components/app/import-token/token-list/index.scss

```diff
@@ -23,7 +23,10 @@
   &__token_component {
     display: flex;
     flex-flow: row nowrap;
-    padding: 16px;
+    padding-left: 16px;
+    padding-right: 16px;
+    padding-top: 8px;
+    padding-bottom: 8px;
```

Comment:
Would it be possible to add these to the `<Box>` instead?

```
paddingLeft={4}
paddingRight={4}
paddingTop={2}
paddingBottom={2}
```

Reply:
[1ff64c5](https://github.com/MetaMask/metamask-extension/pull/26300/commits/1ff64c5e2626b560135d382b4283d5c3490bc9c9)